libs/eavmlib/src/esp.erl

```diff
@@ -279,6 +281,34 @@ deep_sleep() ->
 deep_sleep(_SleepMS) ->
     erlang:nif_error(undefined).
 
+%%-----------------------------------------------------------------------------
+%% @param   Source the device that will be mounted
+%% @param   Target the path where the filesystem will be mounted
+%% @param   FS the filesystem, only fat is supported now
+%% @param   Opts
+%% @returns ok in case of success, otherwise an error
+%% @doc     Mount a filesystem
+%% @end
+%%-----------------------------------------------------------------------------
+-spec mount(
+    Source :: unicode:chardata(),
+    Target :: unicode:chardata(),
+    FS :: fat,
+    Opts :: proplists:proplist() | #{atom() => term()}
+) -> ok.
+mount(_Source, _Target, _FS, _Opts) ->
+    erlang:nif_error(undefined).
+
+%%-----------------------------------------------------------------------------
+%% @param   Target the path of the mounted filesystem that should be unmounted
```

Comment:
Documentation needs to be updated as it's a resource now.

Reply:
Fixed.